config/accounts.go

```diff
@@ -1,9 +1,61 @@
 package config
 
+// RequestType : Request type enumeration
+type RequestType string
+
+// The request types
+const (
+	RequestTypeAMP   RequestType = "AMP"
+	RequestTypeApp   RequestType = "app"
+	RequestTypeVideo RequestType = "video"
+	RequestTypeWeb   RequestType = "web"
+)
```

Comment:
I added this `RequestType` type otherwise there would be a circular dependency if this package `config` was dependent on `pbsmetrics`. I added a map from the `pbsmetrics` `RequestType` to this type elsewhere. Ideally this wouldn't live here. We can find a better place for this but I think there's a larger related issue here regarding package responsibilities that led to the circular dependency issue and this solution. For example, I think the `config` package does too much but perhaps that's a larger team discussion.

Reply:
Good question on where this should live. Perhaps we need a "request constants" or "auction constants" package to resolve.

Reply:
What do you think of calling it 'IntegrationType'? Also.. why is the "AMP" value not lower case like the others?